client/src/localization/languages/De.ts

```diff
@@ -3,78 +3,91 @@
 // file deepcode ignore HardcodedNonCryptoSecret: No hardcoded secrets present in this file
 
 export default {
+  com_error_moderation:
+    'Es scheint, dass der eingereichte Inhalt von unserem Moderationssystem als nicht mit unseren Community-Richtlinien vereinbar gekennzeichnet wurde. Wir können mit diesem spezifischen Thema nicht fortfahren. Wenn Sie andere Fragen oder Themen haben, die Sie erkunden möchten, bearbeiten Sie bitte Ihre Nachricht oder erstellen Sie eine neue Konversation.',
+  com_error_no_user_key: 'Kein Schlüssel gefunden. Bitte geben Sie einen Schlüssel ein und versuchen Sie es erneut.',
```

Comment:
Ich weiß nicht ob "Schlüssel" hier die beste Übersetzung ist. Ich würde sagen "API Key" ist hier sehr viel aussagekräftiger.

/

I don't know if "Schlüssel" is the best translation here. I would say "API key" is much more meaningful here.

Reply:
Changed it to API-Key. Its a common english term used also in German UI widely and thus expected as XHyperDEVX suggests.

Reply:
@XHyperDEVX  can you review it please